server/semantics-service/src/main/resources/templates/freemarker/types.ftl

```diff
@@ -27,8 +27,8 @@
     <p class="text-muted">Each schema defines the expected attributes for a dataset definition.
         The attributes for the <b>${title}</b>
         schema are listed in the table below.</p>
-
-    <table class="table">
+    <a href="/semantics/api/types/${url}/template" class="lead">Download Template</a>
```

Comment:
I think it will also need to have a download property so that it will download the files. It's possible Postman isn't doing it but is there something specific I need to do in Postman? Otherwise if in a browser it works fine then it looks good.

Reply:
I think postman don't have the base hostname capability. I can deploy it in dev cluster shortly.

Reply:
Yes, we can check it properly